cpp/oneapi/dal/algo/kmeans/backend/gpu/train_kernel_lloyd_dense_dpc_test.cpp

```diff
@@ -17,8 +17,8 @@
 #include <CL/sycl.hpp>
```

Comment:
Is this file actually related to PCA?

Reply:
I've added support for DPC++ tests in Bazel to be able to run DPC++ PCA tests and had to fix existing rules/tests.